test/Microsoft.Azure.WebJobs.Host.UnitTests/PublicSurfaceTests.cs

```diff
@@ -155,9 +155,9 @@ public void WebJobsHostPublicSurface_LimitedToSpecificTypes()
                 "ITriggerBindingStrategy`2",
                 "ConnectionStringNames",
                 "JobHost",
-                "JobHostConfiguration",
+                "JobHostOptions",
                 "JobHostInternalStorageConfiguration",
-                "JobHostQueuesConfiguration",
+                "JobHostQueuesOptions",
```

Comment:
Below I see JobHostBlobsConfiguration and other XXXConfiguration types that have been renamed to XXXOptions. Do these tests run?